src/source/Ice/TurnConnection.c

```diff
@@ -399,8 +403,8 @@ STATUS turnConnectionHandleChannelDataTcpMode(PTurnConnection pTurnConnection, P
 
     pCurPos = pBuffer;
     remainingBufLen = bufferLen;
-    while(remainingBufLen > 0) {
-        if (pTurnConnection->currRecvDataLen > 0) {
+    while(remainingBufLen != 0) {
```

Comment:
probably not a big difference, may I know why changing from `>0` to `!=0`?

Reply:
Both are unsigned so won't be less than zero. It's better to be 'precise' in these cases